src/control/Control.Zoom.js

```diff
@@ -81,7 +81,7 @@ export var Zoom = Control.extend({
 
 	_createButton: function (html, title, className, container, fn) {
 		var link = DomUtil.create('a', className, container);
-		link.innerHTML = html;
+		link.innerHTML = '<span aria-hidden="true">'+html+'</span>';

```

Comment:
@Falke-Design in case you didn't notice, the lint check failed:

> src/control/Control.Zoom.js#L84
> Operator '+' must be spaced

Reply:
Changed